terraria-1.4-tmodloader/terraria-1.4-tmodloader.json

```diff
@@ -0,0 +1,229 @@
+      "desc": "Name of the mods folder",
+      "display": "Mods-path",
+      "required": true,
+      "value": "mods"
+    },
+    "npcs": {
+      "type": "string",
+      "desc": "Reduces enemy skipping but increases bandwidth usage. The lower the number the less skipping will happen, but more data is sent. 0 is off.",
+      "display": "npcstream",
+      "required": true,
+      "value": "60"
+    },
+    "password": {
+      "type": "string",
+      "desc": "Server Password - leave blank for none",
+      "display": "Password",
+      "required": true,
+      "value": ""
+    },
+    "players": {
+      "type": "string",
+      "desc": "Max Players",
+      "display": "Players",
+      "required": true,
+      "value": "4"
+    },
+    "port": {
+      "type": "string",
+      "desc": "What port to bind the server to",
+      "display": "Port",
+      "required": true,
+      "value": "7777"
+    },
+    "priority": {
+      "type": "string",
+      "desc": "Set priority 0:Realtime, 1:High, 2:Above-Normal, 3:Normal, 4:Below-Normal, 5:Idle",
+      "display": "priority",
+      "required": true,
+      "value": "1"
+    },
+    "secure": {
+      "type": "string",
+      "desc": "Enable cheat protection = 1 and disable = 0",
+      "display": "secure",
+      "required": true,
+      "value": "1"
+    },
+    "seed": {
+      "type": "string",
+      "desc": "This can not be changed after the world was generated - Seed of the World",
+      "display": "Seed",
+      "required": true,
+      "value": ""
+    },
+    "size": {
+      "type": "string",
+      "desc": "This can not be changed after the world was generated - World Size (1-3 - 3 is biggest)",
+      "display": "World-Size",
+      "required": true,
+      "value": "3"
+    },
+    "sl": {
+      "type": "string",
+      "desc": "Reduces maximum liquids moving at the same time. If enabled may reduce lags but liquids may take longer to settle.",
+      "display": "maximum liquids moving",
+      "required": true,
+      "value": "1"
+    },
+    "upnp": {
+      "type": "string",
+      "desc": "Enable upnp = 1 and disable = 0",
+      "display": "upnp",
+      "required": true,
+      "value": "1"
+    },
+    "version": {
+      "type": "string",
+      "desc": "tModLoader Version",
+      "display": "tModLoader Version (Set this to the latest tModLoader version - see https://github.com/tModLoader/tModLoader/releases/latest)",
+      "required": true,
+      "value": "v2022.05.103.34"
+    },
+    "world": {
+      "type": "string",
+      "desc": "Name of your World",
+      "display": "World-Name",
+      "required": true,
+      "value": "Terraria-World-1"
+    },
+    "wp": {
+      "type": "string",
+      "desc": "Name of the world folder",
+      "display": "Worldpath",
+      "required": true,
+      "value": "worlds"
+    }
+  },
+  "environment": {
+    "type": "tty"
+  },
+  "supportedEnvironments": [
+    {
+      "type": "tty"
+    }
+  ],
+  "requirements": {}
```

Comment:
Please add any requirements needed for this (like what binaries are needed on the server, such as unzip)

Reply:
Added all requirements in https://github.com/PufferPanel/templates/pull/164/commits/f7866d721e4e073d351b147614741dc4fcd72233